modules/ROOT/pages/deployment/storage/s3.adoc

```diff
@@ -0,0 +1,51 @@
+= S3 Simple Storage Service
+:toc: right
+:toclevels: 2
+:description: Infinite Scale can store blobs on S3 storage. This document gives some general information and considerations for the use of S3.
+
+== Introduction
+
+{description} See the xref:deployment/storage/general-considerations.adoc#s3[General Storage Considerations] in the S3 section for which data is stored where.
+
+== S3 Overview
+
+This section gives a brief description about S3 and how it compares to POSIX based filesystems.
+
+S3 based storage is well positioned for customers expecting a very high capacity demand for storing Infinite Scale blobs.
+
+* In S3, objects are stored by name in a collection called a bucket and have no hierarchical order. +
+Objects in a bucket are indexed and can be listed with a prefix based search. This provides a clear scaling advantage. Compared POSIX, the consistency domain is constrained to a single bucket instead of the entire hierarchical tree.
```

Comment:
```suggestion
Objects in a bucket are indexed and can be listed with a prefix based search. This provides a clear scaling advantage. Compared with POSIX, the consistency domain is constrained to a single bucket instead of the entire hierarchical tree.
```